docs/en/pid-eaa-issuance.rst

```diff
@@ -509,8 +509,9 @@ The JWT payload is given by the following parameters:
     * - **authorization_details**
       - Array of JSON Objects. Each JSON Object MUST include the following claims:
 
-            - **type**: it MUST be set to ``openid_credential``,
-            - **credential_configuration_id**: JSON String. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata. For example, in the case of the PID, it MUST be set to ``PersonIdentificationData``.
+            - **type**: REQUIRED. It MUST be set to ``openid_credential``,
+            - **credential_configuration_id**: REQUIRED when ``format`` parameter is not present. JSON String. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata. For example, in the case of the PID, it MUST be set to ``PersonIdentificationData``. It MUST NOT be present if ``format`` parameter is present.
+            - **format**: REQUIRED when ``credential_configuration_id`` parameter is not present. String identifying the format of the Credential requested by the Wallet. It MUST NOT be present if ``credential_configuration_id`` parameter is present.
```

Comment:
This is a breaking change. Current implementation requires only `credential_configuration_id` as a mandatory parameter and `format` parameter is not considered. We will align this in the next milestone.